package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "format": "prettier --write 'src/*.ts'",
     "lint": "prettier --check 'src/*.ts'",
-    "build": "webpack --mode production",
+    "build": "webpack --mode production && cp dist/* vega/static && cp src/vega.js vega/static",
```

Comment:
This copies a bunch of stuff into the static directory for me. Why can't we make the copies we nee apart of the webpack build? For example with https://webpack.js.org/plugins/copy-webpack-plugin/

<img width="368" alt="Screen Shot 2022-01-18 at 20 24 01" src="https://user-images.githubusercontent.com/589034/150046053-83e34fec-ec65-47ce-aef0-8fb57fb8049e.png">

Reply:
From that plugin link:

> ℹ️ copy-webpack-plugin is not designed to copy files generated from the build process; rather, it is to copy files that already exist in the source tree, as part of the build process.

Reply:
Could we make the static folder the target folder for webpack instead of dist? The problem with dist is that it's also the folder where python puts its wheels.

Reply:
I stumbled across this plugin, which can copy files after the build phase